### ISSUE TITLE
Gascraft description: inconsistency in pluralization of "Remnant"

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#3066]
> Centuries ago, **the Remnant built** these ships in order to explore the upper atmosphere of nearby gas giants. The Archons took issue with **the Remnant's** research methods, and the studies were unable to be completed. The remaining gascraft have been gathering dust in storage for centuries while **the Remnant waits** to see if the anger of the Archons will subside.

It's apparent from all other appearances that "Remnant" is plural. 
[The part with an apostrophe is apparently insoluble, but just marking that here.]
If we're following the pattern in pluralization, `the Remnant waits` should be `the Remnant wait` instead.

**Comment by endless-sky** [comment]
I'm actually not sure whether singular or plural makes more sense for the Remnant in general. I've been using a plural when using "the Remnant" as shorthand for "the people of the Remnant," and singular when just talking about "the Remnant" as a faction.

**Comment by endlesssteve** [comment]
Just give my penny worth as native British English speaker.
I don't think there is real difference between "the people of the Remnant" and "the Remnant faction" and as such they should both be plural. Only a organization not directly linked to a specific group of people is singular.
 e.g. the Vatican. the Maffia. the Government  --- singular
the Romans. the Celts. the Toilet cleaners --- plural
However I've played this part several times and didn't even notice. It didn't spoil my game.

**Comment by azerwyn** [comment]
tl;dr -- It doesn't matter.  The dominant, persistent voice(s) will define the inflection norm.

-------
Some words have irregular plurals that do not fit any of the defined grammar rules.  "Proper" use is largely steered by dialectic, cultural, social ear-training -- and thus is fungible.  

**person – people (also persons, in more formal contexts; people can also be a singular noun with plural peoples.)**

Thus Roman and Romans are acceptable with appropriate context -- singular people and plural peoples.
And Maffia and Maffias are also both acceptable with context -- The Formal, the group, group of groups.

You might dare to offend the Italian and Russian Maffias, but don't cross the Cheerleader-Mother Maffia.

The Vatican is the only one of the examples above that is truly singular and that is because it is unique.  It would be a linguistic contortion to refer to the many seats of religious power (Coptic, Orthodox, Catholic, Apostolic) as Christian Vatacans.  However, that is not to say that over time linguistic drift could adopt such an inflection.

Generally, tribal and aboriginal names are not inflected plural; "_that Apache is one of many Apache._"  However, there are exceptions; "_that Aztec is one of many Aztecs_.  Culturally sensitive writing would favor the group's preferred inflection although imperial writing favors the author's or audience's preferred inflection.

Some ethnic or demographic peoples are never inflected plural: Chinese (or other -ese) and Swiss are the most often cited examples.

Loan words from other languages are generally regional and dialectic and may or may not be inflected.

So where does that leave us?  I venture that the Remnant(s) inflection could end up being a running joke if the kibosh isn't put to this question.

(**culture**) Remnant (as a collective whole) are not Roman. == English are not Roman.

(**society**) There are some Remnants that are similar to some Romans (comparing high-born to high-born, or rites-of-passage or courtship rituals).  Surrounding context in needed to establish an understanding of singular or plural.  But my ear tells me that, "_some Remnant are similar to some Roman_" is awkward without additional supporting words.

(**language**) Many Remnants (different dialects and accents) exist in on different planets but they are all Remnant.  == Many Englishes (like fishes) exist in geographic and temporal silos but they are all English.

(**citizenry**) Remnant are not Roman. Remnant are not Romans. Remnants are not Romans. Remnants are not Roman. (all four should be acceptable)

Remnant are a people like English are a people.  Remnant are people like English are people. People are Remnant like people are English.  But are Remnants like English or Englishes?  <**boggle**>
